src/sql/parts/notebook/notebook.css

```diff
@@ -23,4 +23,17 @@
 	text-align: center;
 	display: flex;
 	align-items: center;
+}
+
+.notebookEditor .notebook-info-button {
```

Comment:
How does this compare to our other buttons (e.g. in the SQL editor toolbar, or the ModelViewEditor one)?

Reply:
Extracted some CSS components from Aris extension. We may have to revisit the CSS again. Did not go through SQL toolbar.